src/components/views/messages/ReactionsRowButton.tsx

```diff
@@ -100,27 +105,58 @@ export default class ReactionsRowButton extends React.PureComponent<IProps, ISta
                     content={content}
                     reactionEvents={reactionEvents}
                     visible={this.state.tooltipVisible}
+                    customReactionImagesEnabled={this.props.customReactionImagesEnabled}
                 />
             );
         }
 
         const room = this.context.getRoom(mxEvent.getRoomId());
         let label: string | undefined;
+        let customReactionName: Optional<string>;
         if (room) {
             const senders: string[] = [];
             for (const reactionEvent of reactionEvents) {
                 const member = room.getMember(reactionEvent.getSender()!);
                 senders.push(member?.name || reactionEvent.getSender()!);
+                if (
+                    this.props.customReactionImagesEnabled &&
+                    REACTION_SHORTCODE_KEY.findIn(reactionEvent.getContent())
+                ) {
+                    customReactionName = REACTION_SHORTCODE_KEY.findIn(reactionEvent.getContent());
+                }
```

Comment:
How about something like `customReactionName = this.props.customReactionImagesEnabled ? REACTION_SHORTCODE_KEY.findIn(....) : undefined`